### DISCUSSION TITLE
Did 1.4.2 change the input system?

### DISCUSSION THREAD

**Discussion by ghost** [KadeArchive/Kade-Engine#316] [Q&A] [answered]
I normally beat M.I.L.F with like 30-50 misses, and on 1.4.2, I beat it with 101 misses. So I switched back to 1.4.1 and I beat it with 52. So did the input system get changed?

**Comment by alarmingly** [comment]
yes you idiot it's in the name kade **input engine**

**Reply by isakube** [reply]
oh, boy

1. that's not the name
2. mean
3. they meant inbetween updates 1.4.1 and 1.4.2

**Comment by puyoxyz** [comment, marked as answer]
That's when the new anti mashing system got added(?), so yes, it did change
